### PR TITLE
Add variables for new client DNS variables.

### PR DESCRIPTION
**Pull request by felddy**
# 

## 🗣 Description

Adds to variables that are inserted into the OpenVPN configuration template. 
They send the client the DNS server to use as well as the search domain.

Note: this PR is not to `develop`

## 💭 Motivation and Context

Clients need DNS yo!

## 🧪 Testing

Deployed to production successfully.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -3,6 +3,13 @@
 #
 # You must provide a value for each of these parameters.
 # ------------------------------------------------------------------------------
+variable "client_dns_search_domain" {
+  description = "The DNS search domain to be pushed to VPN clients."
```
Comment (jsf9k) [inline variables.tf:8]: These two variables *must* be specified to be of type `string`.

File: README.md [hunk 2]
```diff
@@ -13,23 +13,21 @@ has been applied
...
-| provisionopenvpn_policy_name | The name to associate with the IAM policy that allows provisioning of OpenVPN in the Shared Services account. | string | `ProvisionOpenvpn` | no |
-| public_zone_name | The name of the public Route53 zone where public DNS records should be created (e.g. "cyber.dhs.gov."). | string | `cyber.dhs.gov` | no |
-| ssm_create_read_role_arn | The ARN of the role to assume when creating a role to allow reading of SSM parameters (e.g. "arn:aws:iam::123456789012:role/SsmCreateReadRole") | string | | yes |
-| tags | Tags to apply to all AWS resources created. | map(string) | `{}` | no |
-| trusted_cidr_blocks | A list of the CIDR blocks outside the VPC that are allowed to access the OpenVPN server (e.g. ["10.10.0.0/16", "10.11.0.0/16"]). | list(string) | `[]` | no |
+|------|-------------|------|---------|:-----:|
+| aws_region | The AWS region where the shared services account is to be created (e.g. "us-east-1"). | `string` | `us-east-1` | no |
```
Comment (jsf9k) [inline README.md:17]: Sigh.

**Review by jsf9k — CHANGES_REQUESTED** [review]
Please specify the type of the two new variables as `string`,

File: variables.tf (hunk 1 above)
Comment (felddy) [reply variables.tf:8]: I think I beat you to it:  a679065

**Review by jsf9k — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (felddy) [reply README.md:17]:
```console
terraform-docs markdown --no-escape --no-header . | sed -E 's/`"|"`/`/g'
```

**Pull request merged**